test/functions_dims.jl

```diff
@@ -55,7 +55,7 @@ end
     end
 end
 
-@testset "permutedims" begin
+@testset "$permutedims" for permutedims in (permutedims, PermutedDimsArray)
```

Comment:
Can we call the function inside this something other than `permutedims` so it doesn't shadow `Base.permutedims`?

Maybe `for permutedim_func in  (permutedims, PermutedDimsArray)`

Reply:
Sure, done! Although I see I went with `f`, same as defn, perhaps less readable...